notificationworkerlambda/cdk/lib/senderworker.ts

```diff
@@ -115,15 +117,37 @@ class SenderWorker extends cdkcore.Construct  {
       reservedConcurrentExecutions: props.reservedConcurrency
     })
 
-    const senderSqsEventSourceMapping = new lambda.EventSourceMapping(this, "SenderSqsEventSourceMapping", {
-      batchSize: props.isBatchingSqsMessages ? 20 : 1,
-      maxBatchingWindow: props.isBatchingSqsMessages ? cdk.Duration.seconds(1) : cdk.Duration.seconds(0),
+    const eventRule = new Rule(this, "SqsMessageSentRule", {
       enabled: true,
-      eventSourceArn: this.senderSqs.queueArn,
-      target: senderLambdaCtr
-    })
-    senderSqsEventSourceMapping.node.addDependency(this.senderSqs)
-    senderSqsEventSourceMapping.node.addDependency(senderLambdaCtr)
+      description: `SQS message sent to SQS queue for ${props.platform} on ${scope.stage}`,
+      ruleName: `${scope.stack}-sender-${props.platform}-${scope.stage}-trigger`,
+      eventPattern: {
+        source: ["com.gu.notifications.worker.Harvester"],
+        detailType: ["Tokens sent"],
+        detail: {
+          platform: [`${props.platform}`],
+        }
+      },
+    });
+    eventRule.addTarget(new LambdaFunction(senderLambdaCtr, {
+      maxEventAge: cdk.Duration.minutes(15), // Optional: set the maxEventAge retry policy
+      retryAttempts: 5, // Optional: set the max number of retry attempts
+    }));
+
+    const scheduledEventRule = new Rule(this, "ScheduledRerunRule", {
+      schedule: Schedule.cron({ minute: "0/10" }),
+    });
```

Comment:
Ideally the EventBridge events trigger our sender lambda which are able to process all SQS messages, but maybe in rare, unexpected cases, a few unprocessed SQS messages are left on the SQS.  I create a scheduled rule to invoke the sender lambda to deal with them if any.